src/nvc.c

```diff
@@ -251,7 +252,42 @@ static int elaborate(int argc, char **argv)
          opt_set_str(OPT_DUMP_VCODE, optarg ?: "");
          break;
       case 'c':
-         opt_set_int(OPT_COVER, 1);
+         if (optarg) {
```

Comment:
Can you split this into a function by itself?

```c
cover_mask_t parse_cover_mask(const char *str)
```

Reply:
Done.

Reply:
But looking at it now, still if we specify explicitly the coverage types, the last one gets skipped. Still need to fix this